javascript-methods/main.js

```diff
@@ -2,8 +2,8 @@ const num1 = 1;
 const num2 = 2;
 const num3 = 3;
 
-const maximumValue = num1 + num2 + num3;
-console.log('Maximum Value:', Math.max(maximumValue));
+const maximumValue = Math.max(num1 + num2 + num3);
```

Comment:
You shouldn't be adding all of your `num` variables together. You should be passing them in as separate arguments